### PR TITLE
adjustment to show Page Type title on SiteTree mouseover

### LINKED ISSUE #2097: Mouseover SiteTree to reveal PageType Behaviour is gone
In SS3 (Possibly v2 also) one could mouseover a SiteTree item and a tooltip would display the PageType. This no longer happens in SS4 - is this by design?

# Notes

 * Track under ORB-175

### PR DESCRIPTION
**Pull request by korthjp17**
fixes #2097

### CODE REVIEW

File: code/Controllers/CMSMain.php [hunk 1]
```diff
@@ -585,6 +585,12 @@ protected function getTreeNodeCustomisations()
                 'listViewLink' => $this->LinkListViewChildren($node->ID),
                 'rootTitle' => $rootTitle,
                 'extraClass' => $this->getTreeNodeClasses($node),
+                'Title' => sprintf(
+                    '(%s: %s) %s',
+                    trim(_t('SilverStripe\Admin\LeftAndMain.PAGETYPE','Page type'), " :"),
```
Comment (dhensby) [inline code/Controllers/CMSMain.php]: You need a space after the commas separating the function arguments:

```diff
-trim(_t('SilverStripe\Admin\LeftAndMain.PAGETYPE','Page type'), " :"),
+trim(_t('SilverStripe\Admin\LeftAndMain.PAGETYPE', 'Page type'), " :"),
```

**Comment by dhensby** [comment]
Thanks for this, only a minor code style issue and then it's good to merge.

This could target 4.0 branch if you wanted it to?

**Comment by dhensby** [comment]
I've rebased this onto 4.0 for you and fixed the code style issue. :)

**Comment by phptek** [comment]
Nice one lads, cheers!

**Pull request merged**